README.md

```diff
@@ -19,6 +19,9 @@ This repository contains the source files (code and documentation) of Ghaf Frame
 
 For information on build instructions and supported hardware, see the [Reference Implementations](https://tiiuae.github.io/ghaf/ref_impl/reference_implementations.html) section of Ghaf documentation.
 
+### OpenSSF Scorecard
+
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/vjuntunen/ghaf-scorecard/badge)](https://securityscorecards.dev/viewer/?uri=github.com/vjuntunen/ghaf-scorecard)
```

Comment:
Based on https://github.com/ossf/scorecard-action?tab=readme-ov-file#scorecard-badge, the Badge and here point to incorrect repo. This points to @vjuntunen 's fork, which I believe is not correct.

This line should be change to:
`[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/tiiuae/ghaf/badge)](https://securityscorecards.dev/viewer/?uri=github.com/tiiuae/ghaf)`

Reply:
fixed